app/facades/foods_facade.rb

```diff
@@ -0,0 +1,15 @@
+class FoodsFacade 
+    def get_foods_search(search)  
+        get_foods_data(search)[:foods].map do |data|
+            Food.new(data)
+        end
+    end 
+
+    def get_foods_data(search)
+        service.get_foods(search)
+    end
+    
+    def service 
+        FoodService.new
```

Comment:
Generally, we would expect Services and Facades to use class methods, so that instances like this are not necessary.